src/App.test.js

```diff
@@ -22,11 +26,31 @@ const response = {
   },
 };
 
+const responseToken = {
+  data: {
+    jwtToken: 'token'
+  },
+};
+
+//Local storage mock
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  removeItem: jest.fn()
+};
+global.localStorage = localStorageMock;
+
 describe('App component', () => {
   afterEach(cleanup);
 
   beforeEach(() => {
-    axios.get = jest.fn(() => Promise.resolve(response));
+    axios.get = jest.fn((url) => {
+      if(url===(process.env.REACT_APP_API_URL + '/Reports/Reports/GetUserData')) {
```

Comment:
Could you fix the spacing between `===` and other symbols here?

Reply:
Does not Prettify or JSLint detect it?

Reply:
Seems he did not run it I suspect...

Reply:
i forgot run. now i run and fixed

Reply:
There are Prettify plugins for most text editors, I do not like them, but maybe you do.